src/Altinn.Notifications.Email/Configuration/AuthSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Altinn.Notifications.Email.Configuration;
+
+/// <summary>
+/// Configuration object used to hold authorization settings for the delivery report controller.
+/// </summary>
+public class AuthSettings
```

Comment:
Did we want to go for the full name here? 

```suggestion
public class AuthorizationSettings
```

Reply:
Changed to EmailDeliveryReportSettings